File: CurriculumRepository.CORE/Data/Models/Scenario/LsDTO.cs
using CurriculumRepository.CORE.Data.Models.Activity;
using CurriculumRepository.CORE.Entities;
using System;
using System.Collections.Generic;

namespace CurriculumRepository.CORE.Data.Models.Scenario
{
    public class LsDTO
    {
        public LsDTO()
        {
            Keywords = new List<string>();
            CorrelationInterdisciplinaritySubjects = new List<string>();
            CollaborationNames = new List<string>();
            TeachingAidUser = new List<TeachingAid>();
            TeachingAidTeacher = new List<TeachingAid>();
            StrategyMethods = new List<StrategyMethodBM>();
            Las = new List<LaDTO>();
        }

        public int Idls { get; set; }
        public string UserId { get; set; }
        public string Lsname { get; set; }
        public string Lsdescription { get; set; }
        public string LstypeName { get; set; }
        public string TeachingSubjectName { get; set; }
        public int? Lsgrade { get; set; }
        public string[] LearningOutcomeSubjects { get; set; }
        public string[] LearningOutcomeCts { get; set; }
        public List<string> Keywords { get; set; }
        public List<string> CorrelationInterdisciplinaritySubjects { get; set; }
        public TimeSpan Lsduration { get; set; }
        public List<StrategyMethodBM> StrategyMethods { get; set; }
        public List<string> CollaborationNames { get; set; }
        public List<TeachingAid> TeachingAidUser{ get; set; }
        public List<TeachingAid> TeachingAidTeacher { get; set; }
        public List<LaDTO> Las { get; set; }
    }
}

File: CurriculumRepository.CLIENT/client-app/src/app/models/scenarios/LsDTO.ts
import { TimeSpan } from '../../helpers/timespan';
import { LaDTO } from '../activity/LaDTO';
import { StrategyMethod } from '../StrategyMethod';
import { TeachingAid } from '../TeachingAid';

export class LsDTO {
    UserId: number;
    Firstname: string;
    Lastname: string;
    Lsname: string;
    Lsdescription: string;
    LstypeName: string;
    TeachingSubjectName: string;
    Lsgrade: number;
    LearningOutcomeSubjects: string;
    LearningOutcomeCts: string;
    CorrelationInterdisciplinaritySubjects: string[];
    Lsduration: TimeSpan;
    StrategyMethods: StrategyMethod[];
    CollaborationNames: string[];
    TeachingAids: TeachingAid[];
    Las: LaDTO[];
}